app/src/main/java/org/wikipedia/language/LanguageUtil.kt

```diff
@@ -144,4 +151,28 @@ object LanguageUtil {
     fun formatLangCodeForButton(languageCode: String): String {
         return languageCode.replace("-", "-\n")
     }
+
+    // TODO: This is a temporary fix for T355192
+    suspend fun getPageSummary(pageSummary: PageSummary?, wikiSite: WikiSite): PageSummary? {
```

Comment:
I'm feeling like a `Util` class should contain completely independent bits of code, with as few dependencies as possible. This feels like it's leaking too much stuff into a Util class. Do you think we could keep this as a private function in `AggregatedFeedContentClient`, to discourage its use in other places?

Reply:
You are right! Done.